src/pages/Title/components/NextSeasons/Stars/Stars.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+//@ts-ignore
+import ReactStars from 'react-rating-stars-component'; // solve the problem with typeScript?
+
+interface StarsProps {
+  value: 0 | 0.5 | 1 | 1.5 | 2 | 2.5 | 3 | 3.5 | 4 | 4.5 | 5;
+  edit: boolean;
+}
+
+export const Stars: React.FC<StarsProps> = ({ value, edit }) => {
+  const ratingChanged = (newRating: number) => {
+    console.log(newRating);
+  };
+
+  return (
+    <div className="stars">
+      <ReactStars
+        value={value}
+        count={5}
+        className="star"
+        onChange={ratingChanged}
+        size={30}
+        isHalf={true}
+        emptyIcon={<i className="far fa-star" />}
```

Comment:
Please do not use Icon font in the project it is old and does not a good approach, find a way with SVG if possible, we will discuss it in depth :)

Reply:
![image](https://github.com/GGO-web/Unimay/assets/97807120/f4ec0b35-51a2-4f9e-9090-8dd2fbdaa7c2)

Reply:
Yes, this is what I wanted to see